client/dashboard/app/router/domains.ts

```diff
@@ -7,6 +8,12 @@ import { domainsQuery } from '../queries/domains';
 import { queryClient } from '../query-client';
 import { rootRoute } from './root';
 
+export const domainsPurchaseRoute = createRoute( {
+	getParentRoute: () => rootRoute,
+	path: 'domains/purchase',
+	component: DomainsPurchase,
```

Comment:
I had to load this synchronously, otherwise the header would show up first, then the overlay. Ideally, we display just the overlay, with the header (below the modal) loading in the background.

But I don't like that. I tried making it async, but it shows up abruptly instead of a smooth fade in. Is there any way to make it fade in nicely, and if possible without manually specifying an `animation` through CSS?

Reply:
You're right, it does. I moved it back to async. However, the purchase component still shows up abruptly. Not sure why it isn't being picked up by the view transition.